src/components/shared/DateAndDebit/DateAndDebit.module.scss

```diff
@@ -0,0 +1,15 @@
+@import "styles/theme";
+
+.layoutDateAndDebit {
+  display: flex;
+  justify-content: space-between;
+  width: inherit;
+  border-top: 1px solid $black-haze;
```

Comment:
You're mixing layout and styles here, please separate these out into separate CSS classes of layout and styles.